compiler/arm/codegen/OMRCodeGenerator.hpp

```diff
@@ -162,6 +162,10 @@ class OMR_EXTENSIBLE CodeGenerator : public OMR::CodeGenerator
    void apply24BitLabelRelativeRelocation(int32_t * cursor, TR::LabelSymbol *);
    void apply8BitLabelRelativeRelocation(int32_t * cursor, TR::LabelSymbol *);
 
+   // ARM specific thresholds for constant re-materialization
+   int64_t getLargestNegConstThatMustBeMaterialized() { TR_ASSERT(0, "Not Implemented on ARM"); return 0; }
```

Comment:
If you are implementing these methods on arm why are these asserting - they should return a sensible value and not assert otherwise why are we creating an implementation?